include/client.h

```diff
@@ -591,19 +589,49 @@ class Client
         Redis* _redis;
 
         /*!
-        *   \brief Execute a Command
-        *   \param cmd The Command to execute
+        *   \brief Execute a non-keyed Command that addresses
+        *          the given db node(s)
+        *   \param cmd The non-keyed Command to execute
+        *   \returns The CommandReply after execution
+        */
+        inline CommandReply _run(AddressAtCommand& cmd);
+
+        /*!
+        *   \brief Execute a non-keyed Command that addresses
+        *          any db node(s)
```

Comment:
Small change, but should this be "Execute a non-keyed Command that addresses any node" since the NonKeyedCommand class only holds one address and port?